lib/notification.js

```diff
@@ -0,0 +1,345 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css, keyframes } from 'styled-components';
+import { UnstyledButton, Button } from './button';
+import { Icon } from './icon';
+import { Progress } from './progress';
+import { code, CodeExample, PropsDefinition, Prop } from './story-utils';
+
+const variants = {
+  notice: css`
```

Comment:
Are these the same colors/font sizes we're using on the community site? The contrast is too low (green is 1.98, purple is 3:1, and red is 2.97. At this font size, the Accessibility Insights tool wants to see 4.5:1 contrast ratios. 

CC: @ttseng, @ehmorris

Reply:
Yeah, there were some tweaks to the notification purple and error red we made a few months ago to improve contrast:

```
notice: #7460E1
error: #DC352C
```

to improve the contrast of `success` I'd suggest lightening the green and changing the text to black:

![image](https://user-images.githubusercontent.com/519336/62897234-d617d500-bd20-11e9-914b-5079b89567a3.png)

(the success green shown above is `#47F68D`

(already discussed with @ehmorris and we're both good with this change)